daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -34,6 +34,9 @@ Maryland 20850 USA.
 
 // q_shared.c -- stateless support routines that are included in each code dll
 #include "q_shared.h"
+
+#include <cstddef>     // for std::size_t
```

Comment:
Haven't noticed `q_shared.h` already includes `stddef.h`.

Does transitive include is clear there and `q_shared` will always has 
'std::size_t' or 'size_t' types imported?